lib/dialects/abstract/utils.js

```diff
@@ -0,0 +1,36 @@
+
+const _dialects = {};
```

Comment:
```
const registry = {}
```

?

Reply:
@jesse23 you called it registries instead of registry, it should be singular as it's a single registry :)